### PR TITLE
Populate log_comment with debug information, /debug/clickhouse route

### PR DESCRIPTION
**Pull request by macobo**
### Changes

This PR:
- Starts populating Clickhouse system.query_log table log_comment column with metadata about the query. This will be useful for debugging performance issues and using clickhouse for query performance analysis.
- Adds a /debug/clickhouse route which shows a table containing information about recent queries by user

image

### Tests
- [ ] Automated tests have been added
- [x] This PR does not require tests

### Changelog
- [x] Entry has been added to changelog
- [ ] This PR does not make a user-facing change

### CODE REVIEW

File: lib/plausible_web/templates/debug/clickhouse.html.heex [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<div class="w-full max-w-4xl bg-white dark:bg-gray-800 shadow-md rounde p-8 mb-4 mt-8 mx-auto">
```
Comment (apata) [inline lib/plausible_web/templates/debug/clickhouse.html.heex:1]: Most trivial suggestion incoming:
```suggestion
<div class="w-full max-w-4xl bg-white dark:bg-gray-800 shadow-md rounded p-8 mb-4 mt-8 mx-auto">
```

**Review by aerosol — COMMENTED** [review]
Nice work!

I attempted to make the query using ecto. If you like it, feel free to cherry-pick https://github.com/plausible/analytics/compare/log-comment-for-query-performance...use-ecto-to-query-logs?expand=1

File: lib/plausible_web/controllers/debug_controller.ex [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            type,
+            event_time,
+            query_duration_ms,
+            query_id,
+            result_rows,
+            formatReadableSize(memory_usage) AS memory_usage,
+            formatReadableSize(read_bytes) AS read_bytes,
+            formatReadableSize(result_bytes) AS result_bytes
+          FROM #{table_expression}
+          WHERE type > 1
```
Comment (aerosol) [inline lib/plausible_web/controllers/debug_controller.ex:52]: Are we ever interested in type != 2 ?

Comment (macobo) [reply lib/plausible_web/controllers/debug_controller.ex:52]: Yep. Getting SQL for errors is also interesting :)

**Review by apata — APPROVED** [review]

File: lib/plausible_web/controllers/debug_controller.ex [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    "result_rows",
+    "memory_usage",
+    "read_bytes",
+    "result_bytes"
+  ]
+
+  def clickhouse(conn, params) do
+    user_id = Map.get(params, "user_id", conn.assigns.current_user.id)
+
+    cluster? = Plausible.MigrationUtils.clustered_table?("events_v2")
```
[the remark this answers is not in the document]
Comment (macobo) [reply lib/plausible_web/controllers/debug_controller.ex]: Good idea, done!

File: lib/plausible_web/controllers/debug_controller.ex [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    "query_duration_ms",
+    "query_id",
+    "result_rows",
+    "memory_usage",
+    "read_bytes",
+    "result_bytes"
+  ]
+
+  def clickhouse(conn, params) do
+    user_id = Map.get(params, "user_id", conn.assigns.current_user.id)
```
[the remark this answers is not in the document]
Comment (macobo) [reply lib/plausible_web/controllers/debug_controller.ex]: Added the ability (via url) but didn't advertise it in the page. We can iterate it in the future.

**Review by aerosol — APPROVED** [review]
🚀

**Pull request merged**